src/ess/dream/__init__.py

```diff
@@ -20,8 +33,35 @@
 
 providers = (*nexus.providers,)
 
+
+def default_parameters() -> dict:
+    # QUantities not available in the simulated data
+    sample = sc.DataGroup(position=sc.vector([0.0, 0.0, 0.0], unit="mm"))
+    source = sc.DataGroup(position=sc.vector([-3.478, 0.0, -76550], unit="mm"))
+    charge = sc.scalar(1.0, unit="µAh")
+    return {
+        RawSample[SampleRun]: sample,
+        RawSample[VanadiumRun]: sample,
+        RawSource[SampleRun]: source,
+        RawSource[VanadiumRun]: source,
+        AccumulatedProtonCharge[SampleRun]: charge,
+        AccumulatedProtonCharge[VanadiumRun]: charge,
+    }
+
+
+def DreamGeant4Workflow() -> sciline.Pipeline:
+    """
+    Workflow with default parameters for the Powgen SNS instrument.
+    """
+    return sciline.Pipeline(
+        providers=powder_providers + geant4_providers, params=default_parameters()
+    )
```

Comment:
- Does this belong into the `__init__.py`?
- Docstring refers to Powgen.

Reply:
> * Does this belong into the `__init__.py`?

hmm in `essans`, we have a separate [workflow file](https://github.com/scipp/esssans/blob/main/src/ess/loki/workflow.py) for loki, in `essreflectometry`, for Amor, we have the [workflow](https://github.com/scipp/essreflectometry/blob/main/src/ess/amor/__init__.py) in the `__init__.py`. We should be consistent and just have a separate workflow file?

> * Docstring refers to Powgen.

Bad copy/paste ;-)

Reply:
I don't think consistency is the main argument, I thought the `__init__.py` should generally be kept minimal?